test/testUtils/mtohUtils.py

```diff
@@ -106,8 +101,15 @@ def setUp(self):
         # so open a new file before each test to minimize leftovers
         # from previous tests.
         mayaUtils.openNewScene()
+        modified = cmds.file(query=True, modified=True)
+        assert not modified, 'Internal test framework error: scene left as modified by mayaUtils.openNewScene()'
+
         self.setHdStormRenderer()
 
+        # We've just opened a new scene, so we should not be modified.  Setting
+        # Storm should conceptually not change that status.
+        cmds.file(modified=False)
```

Comment:
Should this be an assert? The comment makes it sound like this is checking that setting Storm does not change the modified state, but this just forces it to false

Reply:
What confuses me is the contradiction between : 
> [...] we should not be modified. Setting Storm should conceptually not change that status

and
> (setHdStormRenderer()) modifies the file

It's unclear to me whether Storm should mark the file as modified. If it should, then why do force the flag to false, and if it shouldn't, then should this not be an assert, or otherwise point to a problem where setting Storm does change the modified flag without us wanting it to do so?